x/stakeibc/keeper/unbonding_test.go

```diff
@@ -1149,6 +1149,83 @@ func (s *KeeperTestSuite) TestGetUnbondingICAMessages() {
 	}
 }
 
+func (s *KeeperTestSuite) TestBatchSubmitUndelegateICAMessages() {
```

Comment:
Very comprehensive test! 

I don't think there are any meaningful error cases we need to check (the only errors I can think of is the HZ / connection being misconfigured), but just double checking that with you

Reply:
It's a good callout, it definitely wouldn't hurt to add a failure case test

Reply:
done!